src/vex.ts

```diff
@@ -49,13 +49,13 @@ export class Vex {
     return false;
   }
 
-  // Get the 2D Canvas context from DOM element `canvas_sel`.
-  static getCanvasContext(canvasSelector: string): RenderingContext {
-    if (!canvasSelector) {
-      throw new RuntimeError('BadArgument', 'Invalid canvas selector: ' + canvasSelector);
+  // Get the 2D Canvas context from DOM element with id `canvasId`.
+  static getCanvasContext(canvasId: string): RenderingContext {
+    if (!canvasId) {
+      throw new RuntimeError('BadArgument', 'Invalid canvas selector: ' + canvasId);
     }
 
-    const canvas = document.getElementById(canvasSelector) as HTMLCanvasElement;
+    const canvas = document.getElementById(canvasId) as HTMLCanvasElement;
```

Comment:
`canvasId` made more sense to me to use it as parameter of `getElementById` elements can also be selected by class,...

Reply:
Not sure what you are suggesting here.

getElementById can only take a canvas id as a string.

The alternative is to change this to [document.querySelector()](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector), but that is beyond the scope of this PR.

Reply:
My failure, you changed it to `canvasId` It was `canvasSelector`. I thought you changed it to `canvasSelector`.

No modification needed.